common/menu-list.c

```diff
@@ -150,3 +150,53 @@ int menuScan(const char* target) {
     return 0;
 }
 
+
+int themeMenuScan(const char* target) {
+    menuClear();
+    if (chdir(target) < 0) return 1;
+    if (getcwd(s_menu[!s_curMenu].dirname, PATH_MAX+1) == NULL)
+        return 1;
+    DIR* dir;
+    struct dirent* dp;
+    char tmp_path[PATH_MAX+1];
+    dir = opendir(s_menu[!s_curMenu].dirname);
+    if (!dir) return 2;
+
+    while ((dp = readdir(dir)))
+    {
+        menuEntry_s* me = NULL;
+
+        bool shortcut = false;
+        if (dp->d_name[0]=='.')
+            continue;
+
+        memset(tmp_path, 0, sizeof(tmp_path));
+        snprintf(tmp_path, sizeof(tmp_path)-1, "%s/%s", s_menu[!s_curMenu].dirname, dp->d_name);
+
+        const char* ext = getExtension(dp->d_name);
+        char* name = removeExtension(dp->d_name);
```

Comment:
Get rid of this line and the next line + remove the funcs which would then be unused, and just use d_name for the name.

A different name could be specified in the .cfg file, perhaps load it in menuEntryLoad.

Reply:
i was going to add a feature to put a theme name within the config file in the future, and this was going to be a fallback so that if they didnt the filename wouldnt look so ugly with the underscores and extension attached too it... but if you dont want it there ill remove it.

Reply:
removed